index.html

```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="card_section">
```

Comment:
Please ensure that the main container of the projects has a `display:grid` according to the project requirement.